docs/documentation/head/connect.md

```diff
@@ -229,6 +229,15 @@ Connection conn = DriverManager.getConnection(url);
 
     The default is 'true'
 
+* **streamResults** = boolean
+
+    since 42.2.12
```

Comment:
So this should probably. be 43.3.0. It is a breaking change as are a number of others coming up

Reply:
Yeah. this is still WIP. So let's keep this comment visible here until we start considering merging and then fix the number.

Reply:
Fair enough. Also have a look at @vlsi gradle PR too. It will likely be merged in before this.